exercises/practice/rest-api/rest-api.spec.js

```diff
@@ -0,0 +1,153 @@
+import { RestAPI } from './rest-api';
+
+describe('Rest API', () => {
+  describe('user management', () => {
+    test('no users', () => {
+      const restAPI = new RestAPI({ users: [] });
+      expect(restAPI.get('/users')).toEqual({ users: [] });
+    });
+    test('add user', () => {
+      const restAPI = new RestAPI({ users: [] });
+      expect(restAPI.post('/add', { user: 'Adam' })).toEqual({
+        name: 'Adam',
+        owes: {},
+        owed_by: {},
+        balance: 0.0,
```

Comment:
Writing all numbers as `x.0` instead of just `x` does not seems very idiomatic to me since there is no integer vs. float distinction in JavaScript. But it is not wrong in terms of this file making the tests pass of course. :woman_shrugging: